bin/desi_generate_focalplane

```diff
@@ -19,16 +19,20 @@ def main():
                         "devices with an assigned PETAL_ID and DEVICE_LOC "
                         "are considered.")
 
-    parser.add_argument("--collision", type=str, default=None,
-                        required=False,
-                        help="The text config file containing the exclusion "
+    parser.add_argument("--exclusion_file", type=str, default=None,
+                        action="append", required=False,
+                        help="A text config file containing some exclusion "
```

Comment:
Consider using `nargs="*"` instead of `action="append"` to enable
```
--exclusion_file /path/to/fp_settings/collision_settings/*.conf /other/path/collision*.conf ...
```